x-pack/plugin/sql/src/main/resources/org/elasticsearch/xpack/sql/plugin/sql_whitelist.txt

```diff
@@ -32,4 +32,6 @@ class org.elasticsearch.xpack.sql.expression.function.scalar.whitelist.InternalS
   String replace(String, String, String)
   Integer locate(String, String)
   Integer locate(String, String, Integer)
+  Number round(Number, Number)
```

Comment:
The declaration order should match that of the `FunctionRegistry` (the solved one - first based on category, then alphabetically).